src/exported-image-egl.cpp

```diff
@@ -30,6 +30,14 @@
 
 extern "C" {
 
+__attribute__((visibility("default")))
+void
+wpe_fdo_egl_exported_image_set_destroy_notify(struct wpe_fdo_egl_exported_image* image, wpe_fdo_egl_exported_image_destroy_notify_t destroyNotify, void* destroyData)
```

Comment:
This could use an API documentation comment. If we the missing ones little by little with each PR we would incrementally tackle #27 — I wouldn't block merging this PR due to this, though :wink: